### DISCUSSION TITLE
Crate for latest version ?

### DISCUSSION THREAD

**Discussion by spoltier** [visioncortex/vtracer#85] [General]
Thanks @tyt2y3, I'm finding this software very helpful!

I've done some experimenting in Python but would now like to use it in Rust. I see that the latest published crate is for version 0.6.3. It looks like the Python version is already at 0.6.11 which would correspond to the 0.6.5 tag. It would be convenient to have a published crate for that latest tag.

**Comment by TinyBeeman** [comment]
+1!
